word_language_model/main.py

```diff
@@ -211,3 +223,8 @@ def train():
 print('| End of training | test loss {:5.2f} | test ppl {:8.2f}'.format(
     test_loss, math.exp(test_loss)))
 print('=' * 89)
+
+# Export the model in ONNX format.
+print('The model is also exported in ONNX format at {}'.
```

Comment:
put the print inside `export_onnx`, default the script to not `onnx-export`, i.e. default string is  default=''